test/stored_queries/helpers.py

```diff
@@ -4,15 +4,22 @@
 
 _CONF = get_config()
 
-
-def create_test_docs(coll_name, docs):
+def create_test_docs(coll_name, docs, update_on_dupe=False):
     """Create a set of documents for use in tests."""
     body = '\n'.join([json.dumps(d) for d in docs])
+    params = {'overwrite': True, 'collection': coll_name, 'display_errors': '1'}
+
+    if update_on_dupe:
+        del params['overwrite']
+        params['on_duplicate'] = 'update'
```

Comment:
this duplicates the functionality of the DJORNL loader -- I did not want the test db loader to work differently to the scripts that load the Jacobson dataset.